pkg/kubelet/apis/config/validation/validation_test.go

```diff
@@ -38,6 +38,7 @@ var (
 		EnforceNodeAllocatable:          []string{"pods", "system-reserved", "kube-reserved"},
 		SystemReservedCgroup:            "/system.slice",
 		KubeReservedCgroup:              "/kubelet.service",
+		PodLogsRootDirectory:            "/logs",
```

Comment:
I expected to see a check added to ValidateKubeletConfiguration to make sure the path was specified, was absolute, was normalized (so `filepath.Clean(PodLogsRootDirectory) == PodLogsRootDirectory`) and maybe preventing things like spaces in the filename (not sure, will defer to node folks on how fragile the things writing to and reading from that directory are)